frontend/libs/studio-components/src/components/StudioCodelistEditor/StudioCodeListEditor.test.tsx

```diff
@@ -105,32 +107,157 @@ describe('StudioCodeListEditor', () => {
     ]);
   });
 
-  it('Calls the onChange callback with the new code list when a description is changed', async () => {
-    const user = userEvent.setup();
-    renderCodeListEditor();
-    const descriptionInput = screen.getByRole('textbox', { name: texts.itemDescription(1) });
-    const newValue = 'new text';
-    await user.type(descriptionInput, newValue);
-    expect(onChange).toHaveBeenCalledTimes(newValue.length);
-    expect(onChange).toHaveBeenLastCalledWith([
-      { ...codeListWithoutTextResources[0], description: newValue },
-      codeListWithoutTextResources[1],
-      codeListWithoutTextResources[2],
-    ]);
+  describe('onChange without text resources', () => {
```

Comment:
should there be tests for `onChange` being called for `add new row` and `delete row` as well?

Reply:
There are tests for thst already. I considered it was not necessary to make specific tests for each mode (with/without text resources) because those features are independent of that.